cfpb/cfgov-refresh
# Fix for displaying messages when there are no current jobs on Careers pages

Fix for displaying messages when there are no current jobs on Careers pages


## Changes

- Modified `cfgov/jinja2/v1/_includes/organisms/job-listing-list.html` to not display heading when no current jobs.

-  Modifed `cfgov/jinja2/v1/_includes/organisms/table.html` to display message if there are no current jobs. Uses the new `has_data ` and  `empty_table_msg` context properties.

- Modified `cfgov/jobmanager/tests/models/test_blocks.py` to account for new logic and markup when there are no current jobs.

- Modified `cfgov/v1/atomic_elements/organisms.py` to set no `has_data` flag contigent upon the  `first_row_is_table_header` and `first_col_is_header` parameters.

## Testing

-  Run `tox -e fast` and verify that there are no errors.
- Visit `http://localhost:8000/about-us/careers/` and observe the message in the sidebar.
- Visit `http://localhost:8000/about-us/careers/current-openings/` and observe the message in the center of the page.

## Screenshots

- http://localhost:8000/about-us/careers/current-openings/
<img width="984" alt="screen shot 2017-01-09 at 12 03 34 pm" src="https://cloud.githubusercontent.com/assets/1696212/21775340/ae768d94-d663-11e6-9974-3184a6a5b5c7.png">

- http://localhost:8000/about-us/careers/
<img width="448" alt="screen shot 2017-01-09 at 12 04 01 pm" src="https://cloud.githubusercontent.com/assets/1696212/21775354/bbda5eac-d663-11e6-9b0e-a9d45736d056.png">

## Notes
-

## Todos
-

## Checklist
